kcoin/backend.go

```diff
@@ -319,24 +320,13 @@ func (s *Kowala) SetDeposit(deposit uint64) {
 }
 
 func (s *Kowala) StartValidating() error {
-	cb, err := s.Coinbase()
+	_, err := s.Coinbase()
 	if err != nil {
-		log.Error("Cannot start consensus validation without coinbase", "err", err)
+		log.Error("cannot start consensus validation without coinbase", "err", err)
```

Comment:
Is the case of error messages important?

Reply:
you know go guy are very opinionated :)

it's up to us, but golang suggest that

https://github.com/golang/go/wiki/Errors

Reply:
true, worse having mixed, should I revert or change this file?

Reply:
I suppose revert for now and make a do a complete sweep in a different PR?